client/utils.js

```diff
@@ -33,11 +33,13 @@ export const getPathId = () => {
   let pathArr = window.location.pathname.split('/');
   let pathId = 1;
   if (pathArr.length) {
+   
     pathId = parseInt(pathArr.slice(-1)[0]);
-    if (Number.isNaN(pathId) || pathId > 100 || pathId < 1) {
-      pathId = 1;
-    }
+    // if (Number.isNaN(pathId) || pathId > 100 || pathId < 1) {
```

Comment:
Just curious why the removal of this check?

Reply:
since the fec only takes 1 to 100 id, the condition didnt allow the get any data with gameid bigger than 100